Src/BootCamp.Chapter/Program.cs

```diff
@@ -4,9 +4,34 @@ namespace BootCamp.Chapter
 {
     class Program
     {
-        static void Main(string[] args)
+        public static void Main(string[] args)
         {
+            var logger = new Logger();
+            logger.Log("Program boots.");
 
+            for (int num = 0; num < 2; num++)
+            {
+                Console.WriteLine("Calculate BMI for a person.");
+
+                Console.Write("Enter name: ");
+                string name = Console.ReadLine();
+                Console.Write("Enter surname: ");
+                string surname = Console.ReadLine();
+                Console.Write("Enter age: ");
+                int age = Convert.ToInt32(Console.ReadLine());
+                Console.Write("Enter weight: ");
```

Comment:
Use your logger now instead of console :)

Reply:
I have changed the logic to use the logger even though I have concerns about not being able to see the output.